### DISCUSSION TITLE
Improvements in computational efficiency

### DISCUSSION THREAD

**Discussion by vaibhavraghavan** [capytaine/capytaine#557] [General]
Were there improvements made in parallelization/computational speeds as well in Capytaine 2.1?

**Comment by mancellin** [comment]
No, there has not been any specific work on speed in 2.1 (nor in the current development version). The current focus is more accuracy (at constant speed) than performance.

Do you have any specific issue with parallelization and speed?

**Reply by vaibhavraghavan** [reply]
Not really. I previously used v1.4 and I saw significant differences in computational efficient with v2.1. Hence, was curious about the same!
